src/plays/calendar/Calendar.scss

```diff
@@ -0,0 +1,324 @@
+$calendar-box-shadow: 0px 0px 10px 5px rgba(0,0,0,0.1);
+
```

Comment:
We can use `some-unique-class-name` for the CSS Specificity here.

Reply:
@vincentBCP  Please avoid adding a user name in the variable or class in order to make it unique. I would suggest using a combination of words that is unique to the play and also contextually correct.